spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/ExtensionsTestBase.java

```diff
@@ -43,6 +43,8 @@ public static void startMetastoreAndSpark() {
     metastore.start();
     TestBase.hiveConf = metastore.hiveConf();
 
+    TestBase.spark.close();
```

Comment:
you also need to update `TestAggregatePushDown` to do the same. Basically any test that re-configures `spark` from its super class is affected (I believe `TestAggregatePushDown`) is the only other class that's affected

Reply:
I do the fix on this, thank you!